pages.fr/common/apropos.md

```diff
@@ -10,6 +10,6 @@
 
 `apropos -l {{expression_reguliere}}`
 
-- Recherche les pages qui contiennent toutes les expressions données (fonction ET)
+- Recherche les pages qui contiennent toutes les expressions données (fonction ET):
```

Comment:
French needs an space:
```suggestion
- Recherche les pages qui contiennent toutes les expressions données (fonction ET) :
```

Reply:
Oops, I forgot! You're right!  But is that the thin unbreakable space?

Reply:
oh nop.

Reply:
It's this.  I'll edit and apply your suggestion.

Reply:
Obviously the French contributors don't use that char. We should add that to the linter for the French directory.

Reply:
Some do, some don't.  You mean we're able to add non-English language-specific rules to the linter?

Reply:
sounds like a conditional rule based on the parent (of the parent) directory `pages.fr`. I think is possible to add that.

Reply:
Hmm. That's really cool.  But this discussions should get it's own issue because if that's true, we really ought to extend some of the syntactic analysis that the linter can do to some of the other languages, because the punctuation of langs like Spanish/German are very similar to English...

Reply:
I'm currently preparing a PR for normalizing the Chinese punctuation.  They have their own periods, commans, parentheses, semicolons, etc., and their own rules when mixing Latin characters and Chinese characters, etc., and it would be super great for that to be automated.  There's really no need for such tedious tasks to be done over and over again by a human ;)